pkgs/by-name/do/dokieli/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  mkYarnPackage,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  makeWrapper,
+  nodejs,
+}:
+
+mkYarnPackage rec {
+  pname = "dokieli";
+  version = "0-unstable-2024-09-23";
```

Comment:
why the 0 before the version name ?

Reply:
https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning